0.wave1_data/1.cellprofiler_ic_processing/run_loaddata_and_ic.ipynb

```diff
@@ -0,0 +1,1142 @@
+      "Fri Mar 10 15:23:18 2023: Image # 189, module LoadData # 1: CPU_time = 0.56 secs, Wall_time = 0.38 secs\n",
+      "Fri Mar 10 15:23:19 2023: Image # 189, module CorrectIlluminationCalculate # 2: CPU_time = 0.01 secs, Wall_time = 0.00 secs\n",
+      "Fri Mar 10 15:23:19 2023: Image # 189, module CorrectIlluminationCalculate # 3: CPU_time = 0.00 secs, Wall_time = 0.01 secs\n",
+      "Fri Mar 10 15:23:19 2023: Image # 189, module SaveImages # 4: CPU_time = 0.00 secs, Wall_time = 0.00 secs\n",
+      "Fri Mar 10 15:23:19 2023: Image # 189, module SaveImages # 5: CPU_time = 0.00 secs, Wall_time = 0.00 secs\n",
+      "Fri Mar 10 15:23:19 2023: Image # 190, module LoadData # 1: CPU_time = 0.53 secs, Wall_time = 0.36 secs\n",
+      "Fri Mar 10 15:23:19 2023: Image # 190, module CorrectIlluminationCalculate # 2: CPU_time = 0.01 secs, Wall_time = 0.01 secs\n",
+      "Fri Mar 10 15:23:19 2023: Image # 190, module CorrectIlluminationCalculate # 3: CPU_time = 0.01 secs, Wall_time = 0.00 secs\n",
+      "Fri Mar 10 15:23:19 2023: Image # 190, module SaveImages # 4: CPU_time = 0.00 secs, Wall_time = 0.00 secs\n",
+      "Fri Mar 10 15:23:19 2023: Image # 190, module SaveImages # 5: CPU_time = 0.00 secs, Wall_time = 0.00 secs\n",
+      "Fri Mar 10 15:23:19 2023: Image # 191, module LoadData # 1: CPU_time = 0.54 secs, Wall_time = 0.37 secs\n",
+      "Fri Mar 10 15:23:19 2023: Image # 191, module CorrectIlluminationCalculate # 2: CPU_time = 0.01 secs, Wall_time = 0.01 secs\n",
+      "Fri Mar 10 15:23:19 2023: Image # 191, module CorrectIlluminationCalculate # 3: CPU_time = 0.00 secs, Wall_time = 0.00 secs\n",
+      "Fri Mar 10 15:23:19 2023: Image # 191, module SaveImages # 4: CPU_time = 0.00 secs, Wall_time = 0.00 secs\n",
+      "Fri Mar 10 15:23:19 2023: Image # 191, module SaveImages # 5: CPU_time = 0.00 secs, Wall_time = 0.00 secs\n",
+      "Fri Mar 10 15:23:19 2023: Image # 192, module LoadData # 1: CPU_time = 0.65 secs, Wall_time = 0.38 secs\n",
+      "Fri Mar 10 15:23:20 2023: Image # 192, module CorrectIlluminationCalculate # 2: CPU_time = 0.00 secs, Wall_time = 0.01 secs\n",
+      "Fri Mar 10 15:23:20 2023: Image # 192, module CorrectIlluminationCalculate # 3: CPU_time = 0.00 secs, Wall_time = 0.00 secs\n",
+      "Fri Mar 10 15:23:20 2023: Image # 192, module SaveImages # 4: CPU_time = 0.00 secs, Wall_time = 0.00 secs\n",
+      "Fri Mar 10 15:23:20 2023: Image # 192, module SaveImages # 5: CPU_time = 0.00 secs, Wall_time = 0.00 secs\n"
+     ]
+    }
+   ],
+   "source": [
+    "# since we are running an illum pipeline, we do not use `sqlite_name` and `analysis_run`\n",
+    "cp.run_cellprofiler(\n",
+    "    path_to_pipeline=path_to_pipeline,\n",
+    "    path_to_output=path_to_output,\n",
+    "    path_to_loaddata=path_to_loaddata,\n",
+    ")"
+   ]
+  }
+ ],
+ "metadata": {
+  "kernelspec": {
+   "display_name": "pe2loaddata",
+   "language": "python",
+   "name": "python3"
```

Comment:
Is this notebook using the correct environment (looks like this one may have been removed)? If not, consider updating to use the correct environment.

Reply:
After discussion, we decided that we are both unsure about why it is doing this as I did use the `interstellar_data` kernel for the notebook. We will have to keep an eye on this.